doc/tutorials/filesets.rst

```diff
@@ -0,0 +1,350 @@
+============================
+DICOM File-sets and DICOMDIR
+============================
+
+This tutorial is about DICOM File-sets and covers
+
```

Comment:
Yeah, I've been thinking the same thing. I'd like to keep something about DICOMDIR, because that seems to be the entry point that new users know about (rather than File-set), but there's too much focus on it.